packages/driver/src/cypress/error_messages.ts

```diff
@@ -1133,6 +1131,107 @@ export default {
     no_global: `Angular global (\`window.angular\`) was not found in your window. You cannot use ${cmd('ng')} methods without angular.`,
   },
 
+  origin: {
```

Comment:
No changes in this section, just moved the `origin` section to it's correct alphabetical location.